targets/TARGET_STM/TARGET_STM32WB/TARGET_STM32WB5MxG/TARGET_DISCO_WB5MMG/PinNames.h

```diff
@@ -185,10 +192,12 @@ typedef enum {
 } PinName;
 
 // Standardized LED and button names
-#define LED1     PA_7  // LD4 is a RGB LED connected to a PWM LED driver // TODO
+#define LED1     PA_7  // LD4 is a RGB LED connected to a PWM LED driver
```

Comment:
I'd like to understand this a bit more. It seems `LED1` isn't really useful to developers.
In other words, this code won't do anything:
  ```
  DigitalOut myLED(LED1);
  myLED = 1;
  ```
Is this correct?

If so, then we should update the comment and generate a warning at compile time.

Reply:
Not so easy, as this is how RGB driver is using it:
mbed::DigitalOut RGB_LED_SDI(LED1);

https://github.com/ARMmbed/mbed-os/pull/14831/files#diff-ac4d4ed8a6ae01f08b3e54889470d5bae452601085af2e0995559838a6fc256eR66

Reply:
Or we rename LED1 name into RGB_LED ?

Reply:
Would the Greentea test for pin-names work?
https://github.com/ARMmbed/mbed-os/blob/4e121a7d798b859dc06932b62f83c1a69e5d1167/hal/tests/TESTS/pin_names/generic/main.cpp#L37

If we rename, it means the test will generate an error at compile time
https://github.com/ARMmbed/mbed-os/blob/4e121a7d798b859dc06932b62f83c1a69e5d1167/hal/tests/TESTS/pin_names/generic/main.cpp#L28

Reply:
@MarceloSalazar  LED name renamed